course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -97,7 +87,156 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is also possible to operate on cubes with numeric scalars, NumPy arrays and even cube coordinates:"
+    "----\n",
+    "\n",
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Exercise: </font></b>\n",
+    "    <p>Work out which aspects of the cube metadata have changed in the difference result, and print them out.</p>\n",
```

Comment:
It does say what has changed in the cube's metadata in the line above:

> Notice that the resultant cube's name is now `unknown` and that resultant cube's `attributes` and `cell_methods` have disappeared; this is because these all differed between the two input cubes.